packages/theme/stories/theme.stories.ts

```diff
@@ -162,65 +162,4 @@ storiesOf('Theme', module)
                 </div>
             </sp-theme>
         `;
-    })
-    .add('Nested color', () => {
```

Comment:
Can you talk about why you chose to remove this story?

Reply:
The whole story is there to test nesting an `<sp-theme-lightest>` which doesn't exist anymore. There's already another story for nesting an `<sp-theme>`

Reply:
I will put a reversed version in... I am not sure that there is a better alternative to using the stories.